scripts/release/mule/test/test.sh

```diff
@@ -10,35 +11,52 @@ export ARCH_TYPE
 OS_TYPE=$(./scripts/ostype.sh)
 export OS_TYPE
 
-if [ -z "$VERSION" ]; then
-    VERSION=${VERSION:-$(./scripts/compute_build_number.sh -f)}
-fi
-export VERSION
-
-if [ -z "$BRANCH" ]; then
-    BRANCH=${BRANCH:-$(git rev-parse --abbrev-ref HEAD)}
-fi
-export BRANCH
+export BRANCH=${BRANCH:-$(git rev-parse --abbrev-ref HEAD)}
+export CHANNEL=${CHANNEL:-$(./scripts/compute_branch_channel.sh "$BRANCH")}
+export NETWORK=${NETWORK:-$(./scripts/compute_branch_network.sh "$BRANCH")}
+export SHA=${SHA:-$(git rev-parse HEAD)}
+export VERSION=${VERSION:-$(./scripts/compute_build_number.sh -f)}
 
-if [ -z "$CHANNEL" ]; then
-    CHANNEL=${CHANNEL:-$(./scripts/compute_branch_channel.sh "$BRANCH")}
-fi
-export CHANNEL
+PKG_DIR="./tmp/node_pkgs/$OS_TYPE/$ARCH_TYPE"
 
-if [ -z "$SHA" ]; then
-    SHA=${SHA:-$(git rev-parse HEAD)}
-fi
-export SHA
+pushd "$PKG_DIR"
 
 if ! $USE_CACHE
 then
-    mule -f package-test.yaml "package-test-setup-$PKG_TYPE"
+    SRC_DIR="s3://algorand-staging/releases/$CHANNEL/$VERSION"
+
+    # deb
+    aws s3 cp "$SRC_DIR/algorand_${CHANNEL}_${OS_TYPE}-${ARCH_TYPE}_${VERSION}.deb" .
+    aws s3 cp "$SRC_DIR/algorand-devtools_${CHANNEL}_${OS_TYPE}-${ARCH_TYPE}_${VERSION}.deb" .
+
+    # rpm
+    aws s3 cp "$SRC_DIR/algorand-$VERSION-1.$ARCH_BIT.rpm" .
+    aws s3 cp "$SRC_DIR/algorand-devtools-$VERSION-1.$ARCH_BIT.rpm" .
 fi
 
-if [[ "$ARCH_TYPE" =~ "arm" ]]
+popd
+
+for test in $(ls ./scripts/release/mule/test/tests/pre/*.sh)
+do
+    bash "$test"
+done
+
+pushd "$PKG_DIR"
+
+if [ "$PKG_TYPE" = deb ]
 then
-    ./scripts/release/mule/test/tests/run_tests -b "$BRANCH" -c "$CHANNEL" -h "$SHA" -r "$VERSION"
+    dpkg -i algorand_*"$VERSION"*.deb
+    dpkg -i algorand-devtools*"$VERSION"*.deb
 else
-    ./scripts/release/mule/test/util/test_package.sh
+    yum install yum-cron -y
```

Comment:
Is this required because we're installing from package rather than repository?

Reply:
Yes.

Reply:
I just added a comment to clarify.